code_generation/quantities/nanoAOD.py

```diff
@@ -72,3 +73,26 @@
 HTXS_stage1_1_fine_cat_pTjet30GeV = NanoAODQuantity("HTXS_stage1_1_fine_cat_pTjet30GeV")
 HTXS_stage1_2_cat_pTjet30GeV = NanoAODQuantity("HTXS_stage1_2_cat_pTjet30GeV")
 HTXS_stage1_2_fine_cat_pTjet30GeV = NanoAODQuantity("HTXS_stage1_2_fine_cat_pTjet30GeV")
+
+## MET quantities
+## TODO Swich to Puppi versions for METCOV and Signifiance as soon as they are in the nanoAOD
+MET_covXX = NanoAODQuantity("MET_covXX")
+MET_covXY = NanoAODQuantity("MET_covXY")
+MET_covYY = NanoAODQuantity("MET_covYY")
+MET_significance = NanoAODQuantity("MET_significance")
+
+MET_phi = NanoAODQuantity("PuppiMET_phi")
+MET_pt = NanoAODQuantity("PuppiMET_pt")
+MET_sumEt = NanoAODQuantity("PuppiMET_sumEt")
+MET_pt.register_external_shift(
```

Comment:
I think this should be done in the config because analysers may want to include it or maybe not

Reply:
this is reworked now :)